### PR TITLE
Staticcheck

### PR DESCRIPTION
**Pull request by sbinet**
Please take a look.

### CODE REVIEW

**Comment by sbinet** [comment]
@kortschak PTAL.

(could you also remove the `lint` check as being required?)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -92,6 +93,14 @@ jobs:
       run: |
         go test -v ./...
 
+    - name: static-check
```
Comment (kortschak) [inline .github/workflows/ci.yml:96]: I'd like to have this earlier, or potentially running in parallel in it's own job.

If you rename it to be "lint" to match the golangci-lint job, then the branch protection rule doesn't need to be updated.

**Comment by sbinet** [comment]
PTAL

**Review by kortschak — APPROVED** [review]
:tada:

**Pull request merged**